plugins/modules/zos_replace.py

```diff
@@ -0,0 +1,716 @@
+author:
+  - "Marcel Gutierrez (@AndreMarcel99)"
+options:
+  after:
+    description:
+      - If specified, only content after this match will be replaced/removed.
+      - Can be used in combination with I(before).
+    required: false
+    type: str
+  backup:
+    description:
+      - Specifies whether a backup of the destination should be created before
+        editing the source I(target).
+      - When set to C(true), the module creates a backup file or data set.
+      - The backup file name will be returned on either success or failure of
+        module execution such that data can be retrieved.
+    required: false
+    type: bool
+    default: false
+  backup_name:
+    description:
+      - Specify the USS file name or data set name for the destination backup.
+      - If I(src) is a USS file or path, backup_name must be a file
+        or path name, and it must be an absolute path name.
+      - If the source is an MVS data set, I(backup_name) must be an MVS
+        data set name, and the data set must not be preallocated.
+      - If I(backup_name) is not provided, a default name will
+        be used. If the source is a USS file or path, the name of the backup
+        file will be the source file or path name appended with a
+        timestamp, e.g. C(/path/file_name.2020-04-23-08-32-29-bak.tar).
+      - If I(src) is a data set member and backup_name is not provided, the data set
+        member will be backed up to the same partitioned data set with a randomly generated
+        member name.
+      - If it is a Generation Data Set (GDS), use a relative positive name, e.g., I(SOME.CREATION(+1)).
+    required: false
+    type: str
+  before:
+    description:
+      - If specified, only content before this match will be replaced/removed.
+      - Can be used in combination with I(after).
+    required: false
+    type: str
+  encoding:
+    description:
+      - The character set of the source I(target). L(zos_replace,./zos_replace.html)
+        requires it to be provided with correct encoding to read the content
+        of a USS file or data set. If this parameter is not provided, this
+        module assumes that USS file or data set is encoded in IBM-1047.
+      - Supported character sets rely on the charset conversion utility (iconv)
+        version; the most common character sets are supported.
+    required: false
+    type: str
+    default: IBM-1047
+  disable_regex:
+    description:
+      - A list or string that allows the user to specify choices "before", "after", or "regexp" as regular strings instead of regex patterns.
+    required: false
+    type: raw
+  target:
+    description:
+      - The location can be a UNIX System Services (USS) file,
+        PS (sequential data set), member of a PDS or PDSE, PDS, PDSE.
+      - The USS file must be an absolute pathname.
+      - It is possible to use a generation data set (GDS) relative name of generation already.
```

Comment:
Remove period after already
```suggestion
      - It is possible to use a generation data set (GDS) relative name of generation already
```